bin/tag-generator.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+DEFAULT_ALPINE_VERSION = '3.12'
```

Comment:
We should update to a newer base version since 3.12 goes out of support relatively soon.

Reply:
Done.